java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -247,4 +252,68 @@ public void set_missing_value_and_returnOldValue_is_null(BaseClient client) {
         String data = client.set("another", ANOTHER_VALUE, options).get();
         assertNull(data);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void incr_commands_existing_key(BaseClient client) {
+        String key = RandomStringUtils.randomAlphabetic(10);
+
+        assertEquals(OK, client.set(key, "10").get(10, SECONDS));
+
+        assertEquals(11, client.incr(key).get(10, SECONDS));
+        assertEquals("11", client.get(key).get(10, SECONDS));
+
+        assertEquals(15, client.incrBy(key, 4).get(10, SECONDS));
+        assertEquals("15", client.get(key).get(10, SECONDS));
+
+        assertEquals(20.5, client.incrByFloat(key, 5.5).get(10, SECONDS));
```

Comment:
```suggestion
        assertEquals(20.5, client.incrByFloat(key, 5.5).get());
```

Remove the timeout calls everywhere.  They are not needed.

Reply:
I disagree. If something wrong happen, it may hang all rest tests.
An alternate option is to configure junit for test timeout; then we can remove timeouts from `get`.